api/app/Models/User.php

```diff
@@ -141,6 +141,11 @@ public function getIsProAttribute()
         });
     }
 
+    public function getIsBlockedAttribute()
+    {
```

Comment:
blocked_at should be a database field so it's easier to check
blocked history (reason, time of block/unblock) should be in meta though